pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -295,6 +301,88 @@ func TestGetBlockHeader_Negative_Integration(t *testing.T) {
 	}
 }
 
+func TestGetTransactionMerkleProof_Integration(t *testing.T) {
+	transactionHash, err := bitcoin.NewHashFromString(
+		"72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e",
+		bitcoin.InternalByteOrder,
+	)
```

Comment:
`GetTransactionMerkleProof` expects `72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e` hash as the input.

So we can do either `72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e` with `InternalByteOrder`and the resulting hash will not be changed (it will still be `72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e`) or we can do
`4e210df8041914be65ec026f2963c3ae79ff867424c40523edb1adc257fde772` with `ReversedByteOrder` (which will reversed bytes and we will get 72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e`).

I don't think it makes any difference which one we will use as the resulting hash will be the same.
I used the latter, as other integration tests use it.

Reply:
After addressing other review comments, this transaction hash ended up being created like this:
```
	TxHash: hashFromString(
 		"72e7fd57c2adb1ed2305c4247486ff79aec363296f02ec65be141904f80d214e",
 	),
 ```
 in file `keep-core/internal/testdata/bitcoin/transaction.go` and imported into the file with tests.
 See c647f1948545389e01ca9bb8b130e73db55aaf57.